simple_ai_agents/reasoning_agent/.env.example

```
@@ -0,0 +1,53 @@
+﻿# reasoning_agent Environment Configuration
```

dotenv-linter (4.0.0)

[warning] [IncorrectDelimiter] The ﻿# reasoning_agent Environment Configuration key has incorrect delimiter

(IncorrectDelimiter)